samples/SampleOrigin/wwwroot/Index.html

```diff
@@ -0,0 +1,49 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <script type="text/javascript">
+        
+        function createCORSRequest(method, url) {          
+            var request = new XMLHttpRequest();
+            request.open(method, url, true);
+            return request;
+        }
+
+        // Make the CORS request.
+        function makeCORSRequest(method) {
+
+            // Destination server with CORS enabled.
+            var url = 'http://destination.example.com:5000/api';
+            var request = createCORSRequest(method , url);
+
+            if (!request) {
+                alert('CORS not supported');
+                return;
+            }
+            
+            // Response handlers.
+            request.onload = function () {
+                var text = request.responseText;
+                alert('Response from CORS '+method+' request to ' + url + ': ' + text);
+            };
+
+            request.onerror = function () {
+                alert('There was an error making the request for method ' + method);
+            };
+
+            request.send();
+        }
+
+    </script>
+    <title></title>
+</head>
+<body>
+    <form method="post">
```

Comment:
Probably don't need `<form>` tags at all, considering that:

1. They never get submitted
2. They both say `post` even though that doesn't reflect what they each do

(It's fine to have buttons outside of forms in HTML.... I'm pretty sure...)